src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static addToAlive(beast) {
+    this.alive.push(beast);
+  }
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.addToAlive(this);
```

Comment:
```suggestion
    Animal.alive.push(this)
```
You can do it this simple